sourmash/commands.py

```diff
@@ -5,27 +5,36 @@
 
 import argparse
 import csv
-import itertools
-import multiprocessing
+import glob
 import os
 import os.path
 import sys
 import random
+import itertools
+import time
+import collections
+from collections import defaultdict
+from functools import partial
+import numpy as np
+import pathos.multiprocessing as multiprocessing
```

Comment:
can we move this import to line 460, inside the `elif args.input_is_10x:` case? This way we avoid a dependency on `pathos` (if you don't need to use 10x data).

(It used to be like this, but it was changed in this PR)